site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -529,14 +525,14 @@ If successful, you should see a response that looks something like this:
     ],
     "verified": true,
     "tenantId": "31626131-3938-6231-3634-663636656635",
-    "username": "terry@example.com",
+    "username": "jerry@example.com",
```

Comment:
updated to match request body

Reply:
Hmmm. Are you sure about this. I saw the post and it looked like there were two different users created.

Maybe change their names to be something more obvious. We use a lot of the pied piper TV show characters, so dinesh and richard might work.

Reply:
Good call out as there are two users created. I made this update here because the user Terry is created using the FusionAuth UI further up, however in this instance the user Jerry is being created programmatically and this response is to that create request. Jerry's user info is what is shown in the following screenshot of the UI as well.